server/controller/requestController.js

```diff
@@ -0,0 +1,78 @@
+import pg from 'pg';
+import jwt from 'jsonwebtoken';
+
+import developmentConfig from './../config/developmentConfig';
+import testConfig from './../config/testConfig';
+
+let config;
+
+if (process.env.NODE_ENV === 'development') {
+  config = developmentConfig;
+} else {
+  config = testConfig;
+}
+
+const client = new pg.Client(config);
+client.connect();
```

Comment:
Is there a reason why you are creating a client instance, in the request controller? This should be abstracted since you will be using it across your application

Reply:
I am not sure it will work without creating the instance